src/v3/src/components/Widget/index.tsx

```diff
@@ -361,6 +366,17 @@ export const Widget: FunctionComponent<WidgetProps> = (widgetProps) => {
   // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [responseError]);
 
+  // listen to 'hide' event
+  const onHide = useCallback((hideValue: boolean) => {
+    setHide(hideValue);
+  }, []);
```

Comment:
nit: 
since this controls whether it is shown or hidden, I think it would make more sense to refer to the callback as toggling visibility: 
```suggestion
  const toggleVisibility = useCallback((hideValue: boolean) => {
    setHide(hideValue);
  }, []);
```